app/assets/javascripts/discourse/app/mixins/uppy-upload.js

```diff
@@ -71,7 +76,7 @@ export default Mixin.create({
     }
 
     this.set(
-      "uppyInstance",
+      "_uppyInstance",
```

Comment:
I think you could just do `this._uppyInstance = ...` if you are not using uppyInstance in hbs templates you probably don't need to use set.

Reply:
All done!